tests/data/nyaml2nxdl/NXnested_symbols.yaml

```diff
@@ -1,12 +1,12 @@
+category: base
+doc: Test case for verifying handling of symbols inside a nexus class in nested layers of the hierarchy
 symbols:
   doc: teststring
   nfa: Number of fast axes (acquired simutaneously) e.g. emission angle, kinetic energy
   nsa: Number of slow axes (acquired scanning a physical quantity) e.g. lens voltage, photon energy or temperature
   nx: Number of points in the first angular direction
   ne: Number of points in the energy dispersion direction
-category: base
-doc: Test case for verifying handling of symbols inside a nexus class in nested layers of the hierarchy
-NXentry:
+NXentry(NXobject):
```

Comment:
At definition level, a missing superclass could be automatically take NXobject, so the line above could be acceptable as it was: 'NXentry:'
This can be added after the SKM.